src/main/java/com/sforce/ws/bind/TypeMapper.java

```diff
@@ -151,18 +149,22 @@
         return map;
     }
 
-    private final boolean javaTime;
     private String packagePrefix;
     private String interfacePackagePrefix;
+    private final Map<QName, String> xmlJavaMapping;
+    private final Map<String, QName> javaXmlMapping;
 
     public TypeMapper() {
-        this.javaTime = false;
+        boolean javaTime = false;
```

Comment:
Could you please call the constructor using this(null, null, false)?

Reply:
What was the problem with #235? It seems like you would get the right typemapping when you call getXml/JavaMapping no?

Reply:
the problem was that getJavaXmlMapping and getXmlJavaMapping where called too early, before `javaTime` boolean parameter was passed, basically this parameter was ignored.

Reply:
one test failed after that 😕

Reply:
That is such a subtle problem. Good catch.